Ginkgo: Simplify all --focus regexes (#12089)
cilium-team/cilium: Documentation/contributing/testing/e2e.rst

    })

* From the command line: specify a more granular focus if you want to focus on, say, Runtime L7 tests:

::

    ginkgo --focus "Runtime.*L7"

> This will focus on test that contains "Runtime", followed by any

Reviewer: ```suggestion
This will focus on tests that contain "Runtime", followed by any
```

Author: Fixed.

> number of any characters, followed by "L7". --focus is a regular

Reviewer: ```suggestion
number of any characters, followed by "L7". ``--focus`` is a regular
```

Author: Fixed.

> expression and quotes are required is it contains spaces and to escape

Reviewer 2: ```suggestion
expression and quotes are required if it contains spaces and to escape
```

Author: Fixed.

> shell expansion of '*'.

Reviewer: ```suggestion
shell expansion of ``*``.
```

Author: Fixed.
